### ISSUE TITLE
Collapsing a column that has a motion path to a subXsheet has the motion path both inside AND outside the subXsheet

### ISSUE THREAD

**Issue by DarrenTAnims** [opentoonz/opentoonz#4855]
### Description

If you've got a column with a motion path attached to it and collapse it to a subXsheet, you can still see the motion path node on the schematic when you're not inside the subXsheet

### Steps to Reproduce

1. Create a column
2. Attach a motion path to it
3. Highlight this column and any others
4. Collapse to subXsheet.
5. All of the columns selected disappear from the schematic and timeline, leaving the ne subXsheet column and the motion path node!.
6. Open the subXsheet node and you can still see the motion path attached to the column
7. Close the subXsheet and the motion path node is there too.

### Expected Behavior

After collapsing the column with the motion path node, I wouldn't expect to still see the node on the schematic, attached to nothing.
I'd expect to have to open the subXsheet to see the node attached to the column that it's attached to.

### Screenshots, Video & Crash Logs

![ScreenRecording](https://user-images.githubusercontent.com/28575907/234652222-046086e6-c401-4161-9301-3ba28703df80.gif)

### OpenToonz Version

Other (Specify below)

### OpenToonz Version Information

1.7rc

### Operating System

Windows

### GPU

NVIDIA

### Graphics Tablet

None

**Comment by beeheemooth** [comment]
Continuing this theme:
If you "explode the Sub-xsheet" you will get a new "Path" with a different index.
Each new "explosion" will add a new index.
![Screen_0973](https://user-images.githubusercontent.com/25724133/234675865-601fd346-f3fd-4981-a82d-2cbb5f063858.jpg)

**Comment by DarrenTAnims** [comment]
That doesn't surprise me.
When I first spotted this, I expanded the subXsheet and couldn't draw the path in a new path node. Something behind the scenes had clearly gone haywire. (I should have put that in the original report). I had to create ANOTHER path to start using it.

**Comment by shun-iwasawa** [comment]
Thank you for your comment.
I think that the behavior of motion paths not disappearing from the parent Xsheet is at least not a malfunction.
Not only motion paths, but also pegbars behave in the same way.

Even if we make these objects disappear from the parent Xsheet on collapsing, we need to make some exceptions.
For example, if the non-collapsed columns are also connected the motion path, it should not be deleted. And if some value of the deleting pegbar is referenced by another object via expression, a warning should be raised before deleting.
Erasing an object has a larger risk of ruining the scene than adding an object, and should be implemented carefully.
So, I think it will be difficult for me to alter the behavior properly before the v1.7 release.

**Comment by DarrenTAnims** [comment]
It sounds like pegbars ARE different, but as far as I know motion paths can only be connected to 1 column, so if that column is collapsed, it makes sense to not show the path on the main view and only show it when the column it's connected to is shown.  I don't see the point of leaving it loose.
Plus, it seems that the path that is shown outside of the subxsheet is a different one! You CAN delete it. And when you open the subxsheet the REAL path is still connected to the column. So I think it's been duplicated. This feels like a bug. Why are there 2? And is the right one really connected to the column?

**Comment by RodneyBaker** [comment]
The same motion path can be connected to... drive... multiple columns.
It is multiple paths that cannot be connected to the same column.
Perhaps that is what you meant to say.

But...

In the case where multiple columns are driven by the same path if one column is collapsed... a copy or link would need to still be present to still drive the other columns.

**Comment by DarrenTAnims** [comment]
No, I meant to say that I thought you could only connect a path to 1 column.
I'd say that if a column was connected to multiple paths, then you shouldn't be able to collapse just 1 of those columns into a subXsheet. That would be madness to split up a single path like that. But in my case, I DO only have 1 column connected to the path, so there's no reason to leave it at the root too.
In the same way as you have to have all FX nodes connected to collapse them into a group.
